src/services/api.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios"
+
+console.log(process.env.REACT_APP_API_URL);
```

Comment:
It's fine if you wanted to log the endpoint URL for ease of development, but you might want to remove this in production. :)

Reply:
Thanks for your suggestion.